database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,7 +15,7 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $this->down();
-            $table->string('id')->primary();
+            $table->increments('id');
```

Comment:
id is UUID and not incremental string sha @saidabdul80

Reply:
Sorry for touching that

Reply:
No worries, just wanted to clarify